src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1896,15 +2060,10 @@ private EventStream getImageAndContainerEventStream(final EventsParam... eventsP
   }
 
   @SuppressWarnings("deprecation")
-  private void eventStreamAssertions(final Event pullEvent,
-                                     final Event createEvent,
-                                     final Event startEvent,
-                                     final String containerId,
-                                     final String imageName) throws Exception {
-    // pull event assertions
+  private void imagePullEventAssertions(final Event pullEvent) throws Exception {
     assertThat(pullEvent.time(), notNullValue());
     if (dockerApiVersionAtLeast("1.22")) {
-      assertEquals("image", pullEvent.type());
+      assertEquals(IMAGE.getName(), pullEvent.type());
```

Comment:
Perhaps make this method take more parameters like `containerEventAssertions()` below so we don't have to hard-code `IMAGE` and `BUSYBOX_LATEST`?